apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/backend/ast/Ast.kt

```diff
@@ -61,6 +61,7 @@ internal data class CodeGenerationAst(
       val typeRef: TypeRef,
       val schemaTypename: String?,
       val fragmentAccessors: List<FragmentAccessor>,
+      val isTypeCase: Boolean,
```

Comment:
could you pls remind me what is `isTypeCase `?

Reply:
It's a "case" for a "type" condition. I think of it as a response shape under a certain type condition. @martijnwalraven did I get it right?